app/models/grda_warehouse/remote_credential.rb

```diff
@@ -8,9 +8,21 @@ module GrdaWarehouse
   class RemoteCredential < GrdaWarehouseBase
     acts_as_paranoid
     attr_encrypted :password, key: ENV['ENCRYPTION_KEY'][0..31]
+    has_many :external_ids,
+             class_name: 'HmisExternalApis::ExternalId',
+             foreign_key: :remote_credential_id,
+             dependent: :restrict_with_exception
```

Comment:
This and the class methods below should be injected from the driver via an extension.
See https://github.com/degica/rails_drivers#extensions for details.
`drivers/adult_only_households_sub_pop/extensions/grda_warehouse/hud/client_extension.rb` has a simple example. 
You'll need to include `include RailsDrivers::Extensions` in `remote_credentials.rb`